recipes/r-modeltime.resample/meta.yaml

```
@@ -0,0 +1,124 @@
+{% set version = "0.2.3" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-modeltime.resample
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/modeltime.resample_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/modeltime.resample/modeltime.resample_{{ version }}.tar.gz
+  sha256: 7b78fba8338cf01f7f70ea4e254aac96ff0a3d1b04ab45bb9d9047a30b262c5f
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-cli
+    - r-dials
+    - r-doparallel
+    - r-dplyr
+    - r-forcats
+    - r-foreach
+    - r-forecast
+    - r-ggplot2
+    - r-glue
+    - r-gt
+    - r-hardhat >=1.0.0
+    - r-janitor
+    - r-magrittr
+    - r-parallelly
+    - r-parsnip >=0.2.1
+    - r-plotly
+    - r-prophet
+    - r-purrr
+    - r-reactable
+    - r-rlang >=0.1.2
+    - r-scales
+    - r-stanheaders
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-tidymodels
+    - r-tidyverse
+    - r-timetk >=2.8.1
+    - r-workflows >=1.0.0
+    - r-xgboost >=1.2.0.1
+    - r-yardstick >=0.0.8
+    - r-modeltime >=0.3.0
+    - r-rsample
+    - r-recipes
+    - r-crayon
+    - r-progressr
+    - r-tictoc
+    - r-tune
```

Only non-base libraries listed under "Depends" or "Imports" should be specified.

```suggestion
  host:
    - r-base
    - r-cli
    - r-crayon
    - r-dials
    - r-dplyr
    - r-forcats
    - r-ggplot2
    - r-glue
    - r-hardhat
    - r-magrittr
    - r-modeltime >=0.3.0
    - r-parsnip >=0.1.4
    - r-plotly
    - r-progressr
    - r-purrr
    - r-recipes
    - r-rlang >=0.1.2
    - r-rsample
    - r-stringr
    - r-tibble
    - r-tictoc
    - r-tidyr
    - r-timetk >=2.5.0
    - r-tune
    - r-workflows
    - r-yardstick
```